include/uapi/linux/virtio_ids.h

```diff
@@ -46,5 +46,6 @@
 #define VIRTIO_ID_IOMMU        23 /* virtio IOMMU */
 #define VIRTIO_ID_FS           26 /* virtio filesystem */
 #define VIRTIO_ID_PMEM         27 /* virtio pmem */
+#define VIRTIO_ID_DSP          28 /* virtio DSP */
```

Comment:
ADSP?

Reply:
@plbossart I'm open to better naming of course. I guess, yes, some reference to "audio" would be good. I'll take it, thanks.